apps/emqx_authz/src/emqx_authz_api_sources.erl

```diff
@@ -528,3 +475,14 @@ status_metrics_example() ->
                          }
                       ]
      }.
+
+create_authz_file(Body) ->
+    do_update_authz_file(?CMD_PREPEND, Body).
+
+update_authz_file(Body) ->
+    do_update_authz_file({?CMD_REPLACE, <<"file">>}, Body).
+
+do_update_authz_file(Cmd, Body) ->
+    %% API update will placed in `authz` subdirectory inside EMQX's `data_dir`
+    %% ignore `path`
+    update_config(Cmd, maps:without([<<"path">>], Body)).
```

Comment:
nit: If our swagger schema doesn't support the `path` parameter, then we don't need to specifically state that it's not supported here? Just ignore it?